src/sub-blocks/BasicCard/BasicCard.tsx

```diff
@@ -37,7 +45,9 @@ const BasicCard = (props: BasicCardProps) => {
                     )}
                     <Content
                         title={title}
+                        titleId={titleId}
```

Comment:
is it possible to create this ids inside the `Content` component? Do we need some manual id?

Reply:
Yes, but in this case it will be impossible to label and describe `CardBase`, so one had to support both types: ids from props and generated inside Content by default